applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.h

```diff
@@ -151,6 +152,32 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ElasticIsotropic3D
         return StressMeasure_Cauchy;
     }
 
+    /**
+     * @brief Returns the initial stress vector if it is defined in the geometry
+     */
+    const Vector GetInitialStressVector(Parameters& rParameterValues)
+    {
+        const auto &r_geometry = rParameterValues.GetElementGeometry();
+        if (r_geometry.Has(INITIAL_STRESS_VECTOR)) {
+            return r_geometry.GetValue(INITIAL_STRESS_VECTOR);
+        } else {
+            return ZeroVector(this->GetStrainSize());
+        }
+    }
+
+    /**
+     * @brief Returns the initial strain vector if it is defined in the geometry
+     */
+    const Vector GetInitialStrainVector(Parameters& rParameterValues)
```

Comment:
```suggestion
    Vector GetInitialStrainVector(Parameters& rParameterValues) const
```

Reply:
I don't know why if I change this I get a compilation error